src/fuse_api.pxi

```diff
@@ -467,6 +467,25 @@ cdef session_loop_mt(workers):
         stdlib.free(wd)
 
 
+def stop_fuse_from_another_thread():
```

Comment:
That's a rather awkward name. It doesn't match the convention for the other names, and is misleading since you can call the method from any thread. Just call it "stop". Same for the docstring (which doesn't specify what an "other thread" is).

Reply:
I agree the name is pretty poor. My concern was there is already a `close` method which can be confused with a `stop` one (from my understanding, I would say `close` is more a `clean` given is is called after `main` returns, of course renaming `close` is a no-go because it would break the API).

So maybe we should name it `stop_main` to disambiguate ?

Reply:
No, I think `stop` is perfect :-P

Reply:
so be it ^^

Reply:
so be it ^^